source/adapter.js

```diff
@@ -561,6 +561,9 @@ createIceServer = null;
 createIceServers = null;
 //------------------------------------------------------------
 
+//Creates MediaStream object.
+MediaStream = null;
```

Comment:
@miniruwan I think we should not set the `MediaStream` to `null` because this could be defined for other browsers.